docs/args-and-options.md

```diff
@@ -0,0 +1,180 @@
+---
+id: args-and-options
+title: "Args and Options"
+---
+
+`Args[ArgsType]` and `Options[OptionsType]` represent CLI flags. They are descriptions of the process of constructing instances of `ArgsType` and `OptionsType` from a valid input.
+They can be used to construct a `Command[(ArgsType, OptionsType)]` using one of the following methods:
+```scala mdoc:silent
+import zio.cli._
+
+val name = "name"
+val optionsNone = Options.none
+val argsNone = Args.none
+
+Command(name)
+Command(name, optionsNone)
+Command(name, argsNone)
+Command(name, optionsNone, argsNone)
+```
+
+## Args
+`Args[A]` is a description of the process of constructing an instance of `A` from a valid input of the CLI. It is not yet a specified argument for the CLI. In other words, an `Args[A]` defines a collection of valid arguments for a command and a way to construct a value `A` from them.
+
+**ZIO CLI** offers a variety of methods to create basic `Args` that take a string from the user as input and transform it into a value of the corresponding type. 
+```scala mdoc:silent
```

Comment:
Try a bullet-point list, so you can provide a rich description for each one, including the format it expects to find the argument in, and the validation it will perform (if any).